packages/random/random.js

```diff
@@ -35,214 +39,205 @@ var Alea = function () {
-      s0 -= mash(args[i]);
-      if (s0 < 0) {
-        s0 += 1;
-      }
-      s1 -= mash(args[i]);
-      if (s1 < 0) {
-        s1 += 1;
-      }
-      s2 -= mash(args[i]);
-      if (s2 < 0) {
-        s2 += 1;
-      }
+    s1 -= mash(seeds[i]);
+    if (s1 < 0) {
+      s1 += 1;
+    }
+    s2 -= mash(seeds[i]);
+    if (s2 < 0) {
+      s2 += 1;
     }
-    mash = null;
+  }
+  mash = null;
 
-    var random = function() {
-      var t = 2091639 * s0 + c * 2.3283064365386963e-10; // 2^-32
-      s0 = s1;
-      s1 = s2;
-      return s2 = t - (c = t | 0);
-    };
-    random.uint32 = function() {
-      return random() * 0x100000000; // 2^32
-    };
-    random.fract53 = function() {
-      return random() +
-        (random() * 0x200000 | 0) * 1.1102230246251565e-16; // 2^-53
-    };
-    random.version = 'Alea 0.9';
-    random.args = args;
-    return random;
+  const random = () => {
+    const t = (2091639 * s0) + (c * 2.3283064365386963e-10); // 2^-32
+    s0 = s1;
+    s1 = s2;
+    return s2 = t - (c = t | 0);
+  };
 
-  } (Array.prototype.slice.call(arguments)));
+  random.uint32 = () => random() * 0x100000000; // 2^32
+  random.fract53 = () => random() +
+        ((random() * 0x200000 | 0) * 1.1102230246251565e-16); // 2^-53
+
+  random.version = 'Alea 0.9';
+  random.args = seeds;
+  return random;
 };
 
-var UNMISTAKABLE_CHARS = "23456789ABCDEFGHJKLMNPQRSTWXYZabcdefghijkmnopqrstuvwxyz";
-var BASE64_CHARS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ" +
-  "0123456789-_";
+const UNMISTAKABLE_CHARS = '23456789ABCDEFGHJKLMNPQRSTWXYZabcdefghijkmnopqrstuvwxyz';
+const BASE64_CHARS = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ' +
+  '0123456789-_';
 
 // `type` is one of `RandomGenerator.Type` as defined below.
 //
 // options:
 // - seeds: (required, only for RandomGenerator.Type.ALEA) an array
 //   whose items will be `toString`ed and used as the seed to the Alea
 //   algorithm
-var RandomGenerator = function (type, options) {
-  var self = this;
-  self.type = type;
+class RandomGenerator {
+  constructor (type, { seeds = [] } = {}) {
+    this.type = type;
+
+    if (!RandomGenerator.Type[type]) {
+      throw new Error(`Unknown random generator type: ${type}`);
+    }
 
-  if (!RandomGenerator.Type[type]) {
-    throw new Error("Unknown random generator type: " + type);
+    if (type === RandomGenerator.Type.ALEA) {
+      if (!seeds) {
+        throw new Error('No seeds were provided for Alea PRNG');
+      }
+      this.alea = Alea(seeds);
+    }
   }
 
-  if (type === RandomGenerator.Type.ALEA) {
-    if (!options.seeds) {
-      throw new Error("No seeds were provided for Alea PRNG");
+  /**
+   * @name Random.fraction
+   * @summary Return a number between 0 and 1, like `Math.random`.
+   * @locus Anywhere
+   */
+  fraction () {
+    switch (this.type) {
+    case RandomGenerator.Type.ALEA:
+      return this.alea();
+    case RandomGenerator.Type.NODE_CRYPTO: {
```

Comment:
Let's not go with the braces around the `case`es here, as it's not a style we use anywhere else in the Meteor codebase.

(and, IMHO, a `case` should never be enough lines that it warrants braces).

Reply:
This wasn't done because of the multi lines, but because those lines have lexical declarations, which may have unwanted hoisting effects. I.e. this eslint rule: https://eslint.org/docs/rules/no-case-declarations which is both in eslint:recommended and the airbnb style guides.

But in this specific case the brackets add no value, so happy to remove them.